src/chrome/content/rules/airbites.net.ua.xml

```diff
@@ -0,0 +1,9 @@
+<!--
+www.ipv6.airbites.net.ua nonexist
```

Comment:
You don't need to lists hosts that do not exist or that do not support HTTP at all.